docs/content/release_notes/known_issues.md

```diff
@@ -17,9 +17,10 @@ The following severity levels apply to known issues:
 
 |Ticket ID|Severity|Description|
 |---------|--------|-----------|
+|**CSI-5221**|Service|Changing the selector within VolumeGroup YAML does not update the volume group information, such as removing and adding PVCs from the volume group.<br>**Workaround:** Change the selector information from within the PVC file.|
```

Comment:
resolved

Reply:
@samishekh approve

Reply:
@ArbelNathan I think it is better if we write it somehow like this:
operations of adding/removing PVCs will not be found in the volume group information if they were triggered by changing vg selector.
plus I'm not sure about the add, I'll test that.

Reply:
@rivkap this is the right ticket for this CSI-5231

Reply:
updated

Reply:
I can't see why "events" in the description is a `known issue`